nix/pkgs/kld.nix

```diff
@@ -25,33 +26,43 @@ let
   buildInputs = [ openssl ];
   nativeBuildInputs = [ pkg-config ];
   cargoExtraArgs = "--workspace --all-features";
-  outputHashes = {
-    "https://github.com/JosephGoulden/bdk?branch=backport-begin-batch-result" = "sha256-7uK8gVQUk3zFMCu6OxQRKqY3aK39GA+MuAefagSXrtk=";
-    "https://github.com/JosephGoulden/rust-bitcoincore-rpc?branch=jsonrpc" = "sha256-S4Fwm3WAwpddvEz0cIyaIT39PKp4wZrRvJZj6THgt9o=";
-  };
+  # this is a bit of an hack, since we have to copy the vendor dir and find the broken symlink and replace it with the real file
+  # we should remove (or disable) this if pointing to a stable release again
```

Comment:
This is not getting merged to main anyway as we are waiting for the LDK release.